### ISSUE TITLE
Random "Address size fault" crashes

### ISSUE THREAD

**Issue by subesokun** [utmapp/UTM#5498]
**Describe the issue**  
While installing Windows or Ubuntu I often run into the issue that suddenly UTM crashes with a `Address size fault` error. Any idea what the root cause could be or what I'm doing wrong?

<img width="581" alt="image" src="https://github.com/utmapp/UTM/assets/6920421/37b97ad1-bcb1-4b20-b36a-20ff0c9138ec">
<img width="580" alt="image" src="https://github.com/utmapp/UTM/assets/6920421/5ae68589-8b63-4e93-973a-71a89458c63e">

**Configuration**  
* UTM Version: 4.3.2 (beta)
* macOS Version: 13.4.1
* Mac Chip: M1 

**Crash log**  

```
Thread 12 crashed with ARM Thread State (64-bit):
    x0: 0x0000000000000000   x1: 0x0000000000000000   x2: 0x0000000000000000   x3: 0x0000000000000000
    x4: 0x0000000000000000   x5: 0x000000000000002e   x6: 0x0000000000000000   x7: 0x0000000000000001
    x8: 0xdb88efc92d6a0fc4   x9: 0xdb88efc84091ffc4  x10: 0xcccccccccccccccd  x11: 0x000000000000000a
   x12: 0x0000000000000000  x13: 0x0000000000000035  x14: 0x00000000000007fb  x15: 0x0000000093617006
   x16: 0x0000000000000148  x17: 0x00000001f7db8f60  x18: 0x0000000000000000  x19: 0x0000000000000006
   x20: 0x000000016dfbf000  x21: 0x000000000001f063  x22: 0x000000016dfbf0e0  x23: 0x0000000102bc1276
   x24: 0x00000001f0946000  x25: 0x0000000000000000  x26: 0x0000000000000114  x27: 0x0000000000000000
   x28: 0x0000600003ca6e40   fp: 0x000000016dfbe760   lr: 0x00000001982bbc28
    sp: 0x000000016dfbe740   pc: 0x0000000198284724 cpsr: 0x40001000
   far: 0x0000000110b68000  esr: 0x56000080  Address size fault
```

See [utm-win-crash.txt](https://github.com/utmapp/UTM/files/12181155/utm-win-crash.txt) for full crash log.

**Comment by subesokun** [comment]
Another try but same error again. Could it be something related to GL?

```
gl_version 30 - es profile enabled
WARNING: running without ARB/KHR robustness in place may crash
vrend_renderer_fill_caps: Entering with stale GL error: 1280
GLSL feature level 130
```

[utm-crash-debug.log](https://github.com/utmapp/UTM/files/12190760/utm-crash-debug.log)
[utm-crash-log.txt](https://github.com/utmapp/UTM/files/12190761/utm-crash-log.txt)

**Comment by osy** [comment]
Duplicate of #5489

**Comment by subesokun** [comment]
Issue was fixed via #5489 and release [v4.3.4](https://github.com/utmapp/UTM/releases/tag/v4.3.4) is no longer crashing. Thanks a lot!
